### ISSUE TITLE
Got 'Permission denied to generate login hint for target domain'

### ISSUE THREAD

**Issue by M-Younus** [raj-khare/yt-migrate#11]
Whenever I click on 'Choose your old account' I got `Permission denied to generate login hint for target domain`, what might be the issue, I have enabled the youtube data api, I have replace key and ID with my own key and ID. Please help thanks.

**Comment by raj-khare** [comment]
Did you add your domain to authorized javascript origins?

**Comment by raj-khare** [comment]
Can you refer to #6 and lmk if that helps.

**Comment by M-Younus** [comment]
> Did you add your domain to authorized javascript origins?

yes I added, and check for both simple localhost and with port.

**Comment by M-Younus** [comment]
> Can you refer to #6 and lmk if that helps.

yep, Created this issue after following that.

**Comment by raj-khare** [comment]
Most probably it's an "Authorized JS origins" problem. Check if this helps: https://stackoverflow.com/questions/32041418/google-sign-in-website-error-permission-denied-to-generate-login-hint-for-targ/32211878

**Comment by M-Younus** [comment]
> Most probably it's an "Authorized JS origins" problem. Check if this helps: https://stackoverflow.com/questions/32041418/google-sign-in-website-error-permission-denied-to-generate-login-hint-for-targ/32211878

That solves the issue, now one confusion from which account we should create API key and client ID, from old account or new account?Thanks

**Comment by M-Younus** [comment]
> > Most probably it's an "Authorized JS origins" problem. Check if this helps: https://stackoverflow.com/questions/32041418/google-sign-in-website-error-permission-denied-to-generate-login-hint-for-targ/32211878
> 
> That solves the issue, now one confusion from which account we should create API key and client ID, from old account or new account?Thanks

checked with both account, popup appear for login,logged in gave acces but everytime I got 
![image](https://user-images.githubusercontent.com/13608139/86512583-dcbda800-be1c-11ea-92e0-f230219aed18.png)

**Comment by raj-khare** [comment]
I think it doesn't matter. Any account's API key would work.

**Comment by M-Younus** [comment]
> I think it doesn't matter. Any account's API key would work.

right, can you guide about the error I got.

**Comment by raj-khare** [comment]
Is there any setting in your browser that disables popup?

**Comment by M-Younus** [comment]
> Is there any setting in your browser that disables popup?

let me check.

**Comment by M-Younus** [comment]
> > Is there any setting in your browser that disables popup?
> 
> let me check.

Yes, I've to allowed pop ups, and it works.Thanks for this great utility.

**Comment by M-Younus** [comment]
BTW one question, just for check I've exported subscriptions.xml and from that I got I've around 1000 subscriptions, but this extension shows I just have 659 subscriptions, is this know issue...?

**Comment by raj-khare** [comment]
Maybe you have exhausted the quota as you have a huge list (Google provided a very less quota per day). It may take a few days to transfer the complete list.

**Comment by M-Younus** [comment]
also got many errors,
![image](https://user-images.githubusercontent.com/13608139/86515320-84dd6c00-be31-11ea-8dc5-8dca04cc6902.png)

**Comment by M-Younus** [comment]
> Maybe you have exhausted the quota as you have a huge list (Google provided a very less quota per day). It may take a few days to transfer the complete list.

yes we can only subscribe to 80+ channels per day...
but issue is,check the image, this shows that I only have ~660 subscriptions in old channel, but I knew they are around 1000.
![image](https://user-images.githubusercontent.com/13608139/86515342-a2123a80-be31-11ea-8083-c1138730f912.png)

**Comment by M-Younus** [comment]
> also got many errors,
> ![image](https://user-images.githubusercontent.com/13608139/86515320-84dd6c00-be31-11ea-8dc5-8dca04cc6902.png)

is this know error?

**Comment by raj-khare** [comment]
Can you click on the error and share a more detailed error report?

**Comment by raj-khare** [comment]
As per docs, `Quota impact: A call to this method has a quota cost of 1 unit in addition to the costs of the specified resource parts.`, so fetching subs has a cost

**Comment by M-Younus** [comment]
> Can you click on the error and share a more detailed error report?

![image](https://user-images.githubusercontent.com/13608139/86515398-0b924900-be32-11ea-864a-02068150f39b.png)

**Comment by raj-khare** [comment]
That's weird. I'll look into it

**Comment by M-Younus** [comment]
> That's weird. I'll look into it

Ok,Thanks
